client/my-sites/email/email-providers-comparison/in-depth/index.tsx

```diff
@@ -37,7 +37,7 @@ const EmailProvidersInDepthComparison = ( {
 	const selectedSite = useSelector( getSelectedSite );
 
 	const changeIntervalLength = ( newIntervalLength: IntervalLength ) => {
-		if ( selectedSite === null ) {
+		if ( ! selectedSite ) {
```

Comment:
`selectedSite` can be `undefined`, which would not be caught by this null check, so I modified the check to find any falsy value.